RecoVertex/PixelVertexFinding/plugins/alpaka/fitVertices.h

```diff
@@ -75,19 +75,7 @@ namespace ALPAKA_ACCELERATOR_NAMESPACE::vertexFinder {
     alpaka::syncBlockThreads(acc);
     // reuse nn
     for (auto i : cms::alpakatools::uniform_elements(acc, foundClusters)) {
-      bool const wv_cond = (wv[i] > 0.f);
-      if (not wv_cond) {
-        printf("ERROR: wv[%d] (%f) > 0.f failed\n", i, wv[i]);
-        // printing info on tracks associated to this vertex
-        for (auto trk_i = 0u; trk_i < nt; ++trk_i) {
-          if (iv[trk_i] != int(i)) {
-            continue;
-          }
-          printf("   iv[%d]=%d zt[%d]=%f ezt2[%d]=%f\n", trk_i, iv[trk_i], trk_i, zt[trk_i], trk_i, ezt2[trk_i]);
-        }
-        ALPAKA_ASSERT_ACC(false);
-      }
-
+      ALPAKA_ASSERT_ACC(wv[i] > 0.f);
```

Comment:
```suggestion
      if constexpr (verbose) {
        if (not wv[i] > 0.f) {
          printf("ERROR: wv[%d] (%f) > 0.f failed\n", i, wv[i]);
          // printing info on tracks associated to this vertex
          for (auto trk_i = 0u; trk_i < nt; ++trk_i) {
            if (iv[trk_i] != int(i)) {
              continue;
            }
            printf("   iv[%d]=%d zt[%d]=%f ezt2[%d]=%f\n", trk_i, iv[trk_i], trk_i, zt[trk_i], trk_i, ezt2[trk_i]);
          }
        }
      }
      ALPAKA_ASSERT_ACC(wv[i] > 0.f);
```

Reply:
Thanks for the suggestion. I put it in #47132, and I'll close this PR.

PS. In #47132 I used `not (wv[i] > 0.f)` after seeing the following warning.
```bash
In file included from src/RecoVertex/PixelVertexFinding/plugins/alpaka/vertexFinder.dev.cc:13:
src/RecoVertex/PixelVertexFinding/plugins/alpaka/fitVertices.h:79:13: warning: logical not is only applied to the left hand side of this comparison [-Wlogical-not-parentheses]
        if (not wv[i] > 0.f) {
            ^         ~
src/RecoVertex/PixelVertexFinding/plugins/alpaka/fitVertices.h:79:13: note: add parentheses after the '!' to evaluate the comparison first
        if (not wv[i] > 0.f) {
```